dev/integration_tests/ios_add2app/README.md

```diff
@@ -0,0 +1,21 @@
+# iOS Add2App Test
+
+This application demonstrates some basic functionality for Add2App,
+along with a native iOS ViewController as a baseline and to demonstrate
+interaction.
+
+The following functionality is currently implemented:
+
+1. A regular iOS view controller, similar to the default `flutter create` template.
+1. A FlutterViewController subclass that takes over full screen. Demos showing this both from a cold/fresh engine state and a warm engine state.
+1. A demo of pushing a FlutterViewController on as a child view.
+1. A demo of showing both the native and the Flutter views using a platform channel to to interact with each other.
+1. A demo of showing two FlutterViewControllers simultaneously.
+
```

Comment:
Could be useful adding FlutterViewController pushing a native view controller pushing a FlutterViewController and the second FlutterViewController doesn't affect the first one. Feel free to leave a bug and do later.

Reply:
filed https://github.com/flutter/flutter/issues/28342